src/pages/registration/RegistrationFormActions.tsx

```diff
@@ -65,7 +66,11 @@ export const RegistrationFormActions = ({
       );
       dispatch(setNotification({ message: t('feedback.success.update_registration'), variant: 'success' }));
       if (isLastTab) {
-        history.push(getRegistrationLandingPagePath(values.identifier));
```

Comment:
Bugen med "ulagret endringer" er pga endringer her. Må grave mer om hvorfor, og hvordan det best kan løses

Reply:
Ser ut som løsningen er å sikre at form data alltid resettes etter lagring, for å sikre at `dirty` blir satt til false: https://github.com/BIBSYSDEV/NVA-Frontend/pull/5728/commits/e3ceb8e45b5733d004e51175693c0eead29905fe